mimic/model/clb_objects.py

```diff
@@ -16,6 +20,56 @@ def __init__(self):
         self.lbs = {}
         self.meta = {}
 
+    def add_load_balancer(self, tenant_id, lb_info, lb_id, current_timestamp):
+        """Returns response of a newly created load balancer with
```

Comment:
I'm surprised the doc linter didn't complain about this; we typically put docstrings on the line following the opening quote. Given that it got past lint I won't block on it but just for future reference :).